python/ray/tune/tune.py

```diff
@@ -356,20 +327,75 @@ def run_experiments(experiments,
     # because it schematize the experiments
     # and it conducts the implicit registration.
     experiments = convert_to_experiment_list(experiments)
+    trial_executor = trial_executor or RayTrialExecutor(
+        queue_trials=queue_trials,
+        reuse_actors=reuse_actors,
+        ray_auto_init=ray_auto_init)
+
+    runner = TrialRunner(
+        search_alg=search_alg or BasicVariantGenerator(),
+        scheduler=scheduler or FIFOScheduler(),
+        local_checkpoint_dir=experiments[0].checkpoint_dir,
+        remote_checkpoint_dir=experiments[0].remote_checkpoint_dir,
+        sync_to_cloud=sync_to_cloud,
+        checkpoint_period=global_checkpoint_period,
+        resume=resume,
+        launch_web_server=with_server,
+        server_port=server_port,
+        verbose=bool(verbose > 1),
+        trial_executor=trial_executor)
+
+    for experiment in experiments:
+        runner.add_experiment(experiment)
+    return _execute_runner(
```

Comment:
Are we returning the last `ExperimentAnalysis` object, is that intended? Maybe we should return all `ExperimentAnalysis` objects in a list instead.

regardless, can you add to the docstring what we're returning

Reply:
also just realized the `returns` section in the `run` docstring is also `List of Trial objects.`, that sounds wrong

Reply:
fixed